{
    "CVE-ID": "CVE-2023-32269",
    "NVD": {
        "Descriptions": "An issue was discovered in the Linux kernel before 6.1.11. In net/netrom/af_netrom.c, there is a use-after-free because accept is also allowed for a successfully connected AF_NETROM socket. However, in order for an attacker to exploit this, the system must have netrom routing configured or the attacker must have the CAP_NET_ADMIN capability.",
        "Cvss2_BaseScore": 0,
        "Cvss3_BaseScore": 6.7,
        "Cvss2_VectorString": "",
        "Cvss3_VectorString": "CVSS:3.1/AV:L/AC:L/PR:H/UI:N/S:U/C:H/I:H/A:H",
        "CPEs": [
            {
                "URI": "cpe:/o:linux:linux_kernel",
                "FormattedString": "cpe:2.3:o:linux:linux_kernel:*:*:*:*:*:*:*:*",
                "VersionStartExcluding": "",
                "VersionStartIncluding": "",
                "VersionEndExcluding": "6.1.11",
                "VersionEndIncluding": ""
            }
        ]
    },
    "Official": {
        "4.14.306": {
            "source": [
                "https://www.kernel.org/pub//linux/kernel/v4.x/ChangeLog-4.14.306"
            ]
        },
        "4.19.273": {
            "source": [
                "https://www.kernel.org/pub//linux/kernel/v4.x/ChangeLog-4.19.273"
            ]
        },
        "5.10.168": {
            "source": [
                "https://www.kernel.org/pub//linux/kernel/v5.x/ChangeLog-5.10.168"
            ]
        },
        "5.15.93": {
            "source": [
                "https://www.kernel.org/pub//linux/kernel/v5.x/ChangeLog-5.15.93"
            ]
        },
        "5.4.232": {
            "source": [
                "https://www.kernel.org/pub//linux/kernel/v5.x/ChangeLog-5.4.232"
            ]
        },
        "6.1.11": {
            "source": [
                "https://www.kernel.org/pub//linux/kernel/v6.x/ChangeLog-6.1.11"
            ]
        },
        "6.2": {
            "source": [
                "https://www.kernel.org/pub//linux/kernel/v6.x/ChangeLog-6.2"
            ]
        },
        "commit": {
            "netrom: Fix use-after-free caused by accept on already connected socket": [
                {
                    "description": "netrom: Fix use-after-free caused by accept on already connected socket",
                    "commit": "35d5bb094bc7529c15561289a1ea995f897bf2e6",
                    "source": "https://www.kernel.org/pub//linux/kernel/v4.x/ChangeLog-4.14.306",
                    "version": "4.14.306",
                    "upstream_commit": "611792920925fb088ddccbe2783c7f92fdfb6b64",
                    "fixed": [
                        {
                            "fixed_commit": "1da177e4c3f4",
                            "fixed_commit_source": "https://www.kernel.org/pub//linux/kernel/v2.6/ChangeLog-2.6.12",
                            "fixed_commit_version": "2.6.12",
                            "fixed_commit_upstream_commit": []
                        }
                    ]
                },
                {
                    "description": "netrom: Fix use-after-free caused by accept on already connected socket",
                    "commit": "2c1984d101978e979783bdb2376eb6eca9f8f627",
                    "source": "https://www.kernel.org/pub//linux/kernel/v4.x/ChangeLog-4.19.273",
                    "version": "4.19.273",
                    "upstream_commit": "611792920925fb088ddccbe2783c7f92fdfb6b64",
                    "fixed": [
                        {
                            "fixed_commit": "1da177e4c3f4",
                            "fixed_commit_source": "https://www.kernel.org/pub//linux/kernel/v2.6/ChangeLog-2.6.12",
                            "fixed_commit_version": "2.6.12",
                            "fixed_commit_upstream_commit": []
                        }
                    ]
                },
                {
                    "description": "netrom: Fix use-after-free caused by accept on already connected socket",
                    "commit": "dd6991251a1382a9b4984962a0c7a467e9d71812",
                    "source": "https://www.kernel.org/pub//linux/kernel/v5.x/ChangeLog-5.10.168",
                    "version": "5.10.168",
                    "upstream_commit": "611792920925fb088ddccbe2783c7f92fdfb6b64",
                    "fixed": [
                        {
                            "fixed_commit": "1da177e4c3f4",
                            "fixed_commit_source": "https://www.kernel.org/pub//linux/kernel/v2.6/ChangeLog-2.6.12",
                            "fixed_commit_version": "2.6.12",
                            "fixed_commit_upstream_commit": []
                        }
                    ]
                },
                {
                    "description": "netrom: Fix use-after-free caused by accept on already connected socket",
                    "commit": "c27e0eac568a008cdf04ae7e4ea2d3c18717e627",
                    "source": "https://www.kernel.org/pub//linux/kernel/v5.x/ChangeLog-5.15.93",
                    "version": "5.15.93",
                    "upstream_commit": "611792920925fb088ddccbe2783c7f92fdfb6b64",
                    "fixed": [
                        {
                            "fixed_commit": "1da177e4c3f4",
                            "fixed_commit_source": "https://www.kernel.org/pub//linux/kernel/v2.6/ChangeLog-2.6.12",
                            "fixed_commit_version": "2.6.12",
                            "fixed_commit_upstream_commit": []
                        }
                    ]
                },
                {
                    "description": "netrom: Fix use-after-free caused by accept on already connected socket",
                    "commit": "20355b9569bd1fd5a236898524b6dd4117e660d0",
                    "source": "https://www.kernel.org/pub//linux/kernel/v5.x/ChangeLog-5.4.232",
                    "version": "5.4.232",
                    "upstream_commit": "611792920925fb088ddccbe2783c7f92fdfb6b64",
                    "fixed": [
                        {
                            "fixed_commit": "1da177e4c3f4",
                            "fixed_commit_source": "https://www.kernel.org/pub//linux/kernel/v2.6/ChangeLog-2.6.12",
                            "fixed_commit_version": "2.6.12",
                            "fixed_commit_upstream_commit": []
                        }
                    ]
                },
                {
                    "description": "netrom: Fix use-after-free caused by accept on already connected socket",
                    "commit": "5c2227f3f17782d5262ee0979ad30609b3e01f6e",
                    "source": "https://www.kernel.org/pub//linux/kernel/v6.x/ChangeLog-6.1.11",
                    "version": "6.1.11",
                    "upstream_commit": "611792920925fb088ddccbe2783c7f92fdfb6b64",
                    "fixed": [
                        {
                            "fixed_commit": "1da177e4c3f4",
                            "fixed_commit_source": "https://www.kernel.org/pub//linux/kernel/v2.6/ChangeLog-2.6.12",
                            "fixed_commit_version": "2.6.12",
                            "fixed_commit_upstream_commit": []
                        }
                    ]
                },
                {
                    "description": "netrom: Fix use-after-free caused by accept on already connected socket",
                    "commit": "611792920925fb088ddccbe2783c7f92fdfb6b64",
                    "source": "https://www.kernel.org/pub//linux/kernel/v6.x/ChangeLog-6.2",
                    "version": "6.2",
                    "upstream_commit": "",
                    "fixed": [
                        {
                            "fixed_commit": "1da177e4c3f4",
                            "fixed_commit_source": "https://www.kernel.org/pub//linux/kernel/v2.6/ChangeLog-2.6.12",
                            "fixed_commit_version": "2.6.12",
                            "fixed_commit_upstream_commit": []
                        }
                    ]
                }
            ]
        }
    }
}